numpy/core/src/umath/logical_gufuncs.h

```diff
@@ -0,0 +1,13 @@
+typedef PyObject* (*PyUFunc_FromFuncAndDataAndSignature_t)(PyUFuncGenericFunction*,
```

Comment:
I'm by no means an expert, but I think that by analogy with all the other routines, you'd need to start this with:
```
#ifndef _NPY_UMATH_LOGICAL_GUFUNCS_H_
#define _NPY_UMATH_LOGICAL_GUFUNCS_H_
```
and then end with
```
#endif
```
(I don't really know if you have to worry about this being included more than once, but perhaps better safe than sorry?)